src/server/services/about/views/service.test.js

```diff
@@ -0,0 +1,214 @@
+  function passThrough(item) {
+    return item
+  }
+
+  it('renders the page for a frontend service', () => {
+    const rendered = nunjucksTestEnv.render(
+      'services/about/views/service.njk',
+      {
+        getAssetPath: passThrough,
+        routeLookup: passThrough,
+        tabDetails: {
+          label: 'Service tabs',
+          tabs: [
+            {
+              isActive: true,
+              url: 'services/my-frontend-service',
+              label: 'About'
+            }
+          ]
+        },
+        pageTitle: 'my-frontend-service microservice',
+        summaryList: [],
+        vanityUrls: [
+          {
+            environment: 'infra-dev',
+            urls: [
+              {
+                shuttered: false,
+                enabled: true,
+                url: 'https://my-frontend-service.infra-dev.microservice.com'
+              },
+              {
+                shuttered: true,
+                enabled: true,
+                url: 'https://cdp-something-else.microservice.com'
+              }
+            ]
+          },
+          {
+            environment: 'dev',
+            urls: [
+              {
+                shuttered: false,
+                enabled: true,
+                url: 'https://my-frontend-service-dev.com'
+              }
+            ]
+          }
+        ],
+        apiGateways: [],
+        service: {
+          serviceName: 'my-frontend-service',
+          isFrontend: true
+        },
+        isServiceOwner: false,
+        environmentsWithADeployment: ['infra-dev', 'dev'],
+        environments: ['infra-dev', 'dev', 'test', 'prod'],
+        runningServices: [
+          {
+            environment: 'infra-dev',
+            unstable: false,
+            status: 'running',
+            service: 'my-frontend-service',
+            version: '0.0.2',
+            cdpDeploymentId: 'abc12345',
+            updated: '2025-02-10 12:34',
+            statusClassName: 'govuk-tag--green'
+          },
+          {
+            environment: 'dev',
+            unstable: true,
+            status: 'pending',
+            service: 'my-frontend-service',
+            version: '0.0.2',
+            cdpDeploymentId: 'def9867436',
+            updated: '2025-02-11 14:54',
+            statusClassName: 'govuk-tag--purple'
+          }
+        ],
+        latestPublishedImageVersions: [
+          {
+            tag: '0.0.1',
+            created: '2025-01-25 10:00:00'
+          },
+          {
+            tag: '0.0.2',
+            created: '2025-01-27'
+          },
+          {
+            tag: '0.0.3',
+            created: '2025-01-28'
+          }
+        ],
+        breadcrumbs: [
+          {
+            text: 'Services',
+            href: '/services'
+          },
+          {
+            text: 'my-frontend-service'
+          }
+        ]
+      }
+    )
+    expect(rendered).toMatchFile(undefined, { fileExtension: '.html' })
```

Comment:
this feels a bit weird, having to always add in the `fileExtension` options. Would be worth talking through why